src/main/resources/feedbackQuestionConstSumSubmissionFormOptionFragment.html

```diff
@@ -1,7 +1,7 @@
-<div class="form-group">
-    <label class="col-sm-4 col-md-2 control-label" ${constSumOptionVisibility}>${constSumOptionValue}</label>
-    <div class="col-sm-6 col-md-3">
-        <input type="number" class="form-control pointsBox" ${disabled} 
+<div class="margin-bottom-15px flex">
```

Comment:
I understand that ```form-group``` does nothing (I searched for it and do not see it - correct me if I'm wrong), leave it there and instead add an issue to have it removed entirely